ext/soap/tests/custumheader.phpt

```diff
@@ -0,0 +1,70 @@
+--TEST--
+SOAP customized Content-Type, eg. SwA use case
+--SKIPIF--
+<?php
+	require_once('skipif.inc');
+
+	if (!file_exists(dirname(__FILE__) . "/../../../sapi/cli/tests/php_cli_server.inc")) {
+		echo "skip sapi/cli/tests/php_cli_server.inc required but not found";
+	}
+?>
+--FILE--
+<?php
+
+include dirname(__FILE__) . "/../../../sapi/cli/tests/php_cli_server.inc";
+
+$router = "custh_stubserver.php";
+$args = substr(PHP_OS, 0, 3) == 'WIN' ? "-d extension_dir=" . ini_get("extension_dir") . " -d extension=php_soap.dll" : "";
+$code = <<<'PHP'
+/* Receive */
+$content = trim(file_get_contents("php://input")) . PHP_EOL;
+
+$vars = serialize($_SERVER);
+
+/* record the headers */
+file_put_contents("custh_dump.log", $vars);
+//file_put_contents("custh_dump.log", $content, FILE_APPEND);
```

Comment:
debug leftover?

Reply:
No it is not debug ; custh_dump.log is used to provide back the HTTP headers  from the server to the phpt test so we can check we have the proper headers.

The headers are serialized then they are restore as an array, then they are parsed.

Reply:
Fixed by 04c2262